python/pyarrow/tests/strategies.py

```diff
@@ -29,6 +30,11 @@
     import zoneinfo
 except ImportError:
     zoneinfo = None
+if sys.platform == 'win32':
+    try:
+        import tzdata
+    except ImportError:
+        tzdata = None
```

Comment:
Potential idea to simplify this: in this case we could also set `zoneinfo = None`, and keep the below code similar (then we only need to check `zoneinfo` to see if the test should be skipped or not). We could then just update the skip message to something like "no module named zoneinfo (or tzdata on Windows)"

Reply:
Yes, thought about this too! Will do 👍